lib/Locale/Country/Extra.pm

```diff
@@ -33,15 +52,13 @@ sub country_from_code {
 sub code_from_country {
     my ($self, $country) = @_;
 
-    my %code_countries = reverse %{ $self->_country_codes };
+    $country = lc $country;
+    return $COUNTRY_MAP{$country} if $COUNTRY_MAP{$country};
```

Comment:
So this can also overwrite countries added to Locale::Country::Multilingual is there any consequence of this and should it be documented.?

Reply:
this change is totally compatiable as pervious feature, and not affected other  module or subs.
the document is added in below pod and readme, also many related test cases